data/dialog phrases.txt

```diff
@@ -43,6 +43,10 @@ phrase "generic cargo on visit"
 	word
 		`You have reached <planet>, but not all of the cargo is in the system! Better depart and wait for your escorts to arrive in this star system.`
 
+phrase "generic passenger stopover pickup"
+	word
+		`Your <passengers> board your ship, ready to go to <planet>.`
```

Comment:
Right, you've pointed this out before, I'm just not sure how to address it.  Are there other missions that deal with possible singular passengers that we can look to for an example?  Or should we set the minimum to 2 to avoid this?

Reply:
https://github.com/endless-sky/endless-sky/blob/14183eaeab1b95d4c9f42119257f882d0bda6bc9/data/dialog%20phrases.txt#L48

It looks like they just word it so that passenger count doesn't matter. That may be difficult to do and keep this one's meaning though...

What about changing the one mission that uses this phrase incorrectly to have the words custom-defined, instead of using a generic phrase?
Comment below.

(also, apologies that I've brought it up before - I tend to be a bit forgetful about these things)

Reply:
No worries, it's worth bringing up again.  It's so buried it may look like I missed or forgot about it.

Having taken a closer look, "recruiter" would probably work for the mission, since it's specifically only for one recruiter.

Reply:
This phrase is also used for two other recruiter return missions, which both have multiple passengers.

Reply:
Okay, good, so I'm not leaving a dangling phrase with the edit I just made!  Maybe those should be put to a minimum of 2 if they aren't already.